package chapter.six.three.three.one

import org.junit.jupiter.api.Test

internal class ExceptionTestTest {
    @Test
    fun maintest() {
        ExceptionTest.main(arrayOf())
    }
}